docs/04-FunctionalTests.md

```diff
@@ -250,10 +267,43 @@ error_level: "E_ALL & ~E_STRICT & ~E_DEPRECATED"
 
 `error_level` can also be set globally in `codeception.yml` file.
 
+## Accessing external URL's with Guzzle
```

Comment:
Sorry, this is not a valid part for Functional tests.
1. Functional tests are executed inside a framework with no external calls. 
2. You are trying to reimplement PhpBrowser (which is described in Acceptance Testing chapter)
3. Yes, you can use Guzzle but in most cases you need to make some custom HTTP call to 3rd party you should place this in a Helper and use like: `$I->sendCustomRequest()`. 
4. Mixing scenario-driven `$I->` style with plain PHP code is only applicable for unit testing; However, you are not testing Guzzle library here. 

I hope this explains philosophy a bit